module.json

```
@@ -119,24 +132,25 @@
         "readOnly": true
       }
     },
-    "Next Event": {
+    "messageTimer": {
```

This is why line 77 complains 

```suggestion
    "Next Event": {
```